python/lib/exifedit.py

```diff
@@ -34,7 +35,7 @@ def create_mapillary_description(filename, username, email,
     heading = exif.extract_direction()
     if heading is None:
         heading = 0.0
-    heading = normalize_bearing(interpolated_heading) if interpolated_heading is not None else normalize_bearing(heading)
+    heading = normalize_bearing(interpolated_heading + offset_angle) if interpolated_heading is not None else normalize_bearing(heading + offset_angle)
```

Comment:
Can you comment more on why you added this `normalize_bearing` code in e0978c5 ?

I feel like this code would break under whatever situation the original bit manipulation code was added for:
```
heading = (interpolated_heading + offset_angle) if interpolated_heading is not None else (heading + offset_angle)
heading = normalize_bearing(heading%360)
```